Artifacts/windows-visualstudio/artifact.ps1

```diff
@@ -0,0 +1,122 @@
+Param(
+    [Parameter(Mandatory=$true)]
+    [ValidateSet("2015","2017")] 
+    [string] $version,
+
+    [Parameter(Mandatory=$true)]
+    [ValidateSet("Professional","Enterprise")] 
+    [string] $sku,
+
+    [Parameter()]
+    [string] $installerArgs
+)
+
+function DownloadToFilePath ($downloadUrl, $targetFile)
+{
+    Write-Output ("Downloading installation files from URL: $downloadUrl to $targetFile")
+    $targetFolder = Split-Path $targetFile
+
+    if((Test-Path -path $targetFile))
+    {
+        Write-Output "Deleting old target file $targetFile"
+        Remove-Item $targetFile -Force | Out-Null
+    }
+
+    if((Test-Path -path $targetFolder) -eq $false)
```

Comment:
The following seems more natural, but very subjective, of course:
```
if (-not (Test-Path -path $targetFolder))
```

Reply:
Either way is fine with me.